lib/event.rb

```diff
@@ -0,0 +1,67 @@
+require "date"
+
+class Event
+  attr_reader :name, :food_trucks
+
+  def initialize(name)
+    @name = name
+    @food_trucks = []
+  end
+
+  def add_food_truck(food_truck)
+    @food_trucks << food_truck
+  end
+
+  def food_truck_names
+    @food_trucks.map(&:name)
+  end
+
+  def food_trucks_that_sell(item)
+    @food_trucks.filter do |food_truck|
+      food_truck.check_stock(item).positive?
+    end
```

Comment:
This works! I don't see filter used much in Ruby, but it's the same as select and I know JS uses filter. Nice job.